src/frontends/onnx/README.md

```diff
@@ -0,0 +1,54 @@
+# ONNX FE
+
+The main responsibility of ONNX Frontend is importing ONNX models and conversion them to the `ov::Model` representation. 
+To capabilities of ONNX Frontend belong also:
+* modification of tensors properties (like data type and shapes)
+* changing topology of the models (like cutting subgraphs, inserting additional inputs and outputs)
+* user-friendly searching the models (via tensors and operators names)
+
+The component is written in `C++`, but `Python` bindings are also available.
+Each change to ONNX FE component requires `clang-format` code style and `NCC` naming style checks.
+
+
+## Key contacts
+
+In case of any questions, review and merge requests, please contact:
+* Tomasz Dołbniak (tomdol)
+* Mateusz Tabaka (mateusztabaka)
+* Mateusz Bencer (mbencer)
+
+from `openvino-onnx-frontend-maintainers` maintainers group.
+
+
+## Components
+
+ONNX Frontend implements an interface common for all frontends defined in [Frontends API](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/common/include/openvino/frontend).
+The exported symbols are decorated with `ONNX_FRONTEND_API` flag. The key class responsible for importing is `FrontEnd` defined in (ONNX Frontend class)[https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/include/openvino/frontend/onnx/frontend.hpp].
+For backward compatibility reasons, the ONNX importer API (more lower-level abstraction approach) is still maintained. It can be found in (ONNX Importer)[https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/include/onnx_import/onnx.hpp]. The symbols of ONNX Importer API are decorated with `ONNX_IMPORTER_API` flag.
+
+The crucial place in ONNX Frontend is (ops implementation folder)[https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/frontend/src/op], where the operators are implemented. Each built-in operator has to be registered in ops bridge - (ops bridge)[https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/ops_bridge.cpp]. Expect that, ONNX frontend gives capabilities to register a custom op by an user. It can be achieved via `ConversionExtension` available in (Conversion)[https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/include/openvino/frontend/onnx/extension/conversion.hpp].
```

Comment:
```suggestion
The crucial place in ONNX Frontend is the directory where the operators are implemented:[https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/frontend/src/op]. Each operator handler has to be registered in ops bridge - (ops bridge)[https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/ops_bridge.cpp]. Expect that ONNX frontend has capabilities to register a custom op by a user. It can be achieved via `ConversionExtension` available in (Conversion)[https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/include/openvino/frontend/onnx/extension/conversion.hpp].
```

Reply:
done